Ginger/Ginger/Actions/ActionConversion/SelectActionWzardPage.xaml.cs

```diff
@@ -96,7 +138,7 @@ private void SetGridView()
 
             view.GridColsView.Add(new GridColView() { Field = nameof(ConvertableActionDetails.Selected), Header = "Select", WidthWeight = 3.5, MaxWidth = 50, StyleType = GridColView.eGridColStyleType.CheckBox, BindingMode = System.Windows.Data.BindingMode.TwoWay });
             view.GridColsView.Add(new GridColView() { Field = nameof(ConvertableActionDetails.SourceActionTypeName), WidthWeight = 15, Header = "Source Action Type" });
```

Comment:
Why source Activity was removed? 
we need to show the Activities containing those Action types, you need to agragarte all Activities relevent in list like "Activity1, Activity2,..." and show it

BTW- this Source Colum should not be shown in multi BF's convertion

Reply:
implemented